tests/autoyast/installation.pm

```diff
@@ -122,6 +122,11 @@ sub verify_timeout_and_check_screen {
     return check_screen $needles, $check_time;
 }
 
+sub untick_welcome_on_next_startup {
+    assert_and_click 'dont-show-on-next-startup';
+    assert_and_click 'close-welcome';
+}
```

Comment:
To be reusable, this should be in a shared lib.
I think the best match would be **lib/x11utils.pm**